components/MentorFormRegistration.js

```diff
@@ -378,7 +409,107 @@ export default function MentorForm() {
               value={formData.description}
             />
           </div>
-          <div style={{ gridColumn: "1/3" }}>
+          <div>
+            <label htmlFor="priceSession">Book session Date picker day</label>
+            <input
+              type="text"
+              name="priceSession"
+              className="mentorFormInput"
+              onChange={(e) => updateDatePickerDay(e)}
+              placeholder="e.g. Monday,Sunday,Friday"
+              required
+              value={formData.bookSession[0].datePickerDay?.join(",")}
+            />
+          </div>
+          <div>
+            <label htmlFor="priceSession">Book session Date picker time</label>
+            <input
+              type="text"
+              name="priceSession"
+              className="mentorFormInput"
+              onChange={(e) => updateDatePickerTime(e)}
+              required
+              placeholder="e.g. 11:05AM,03:30PM"
+              value={formData.bookSession[0].datePickerTime?.join(",")}
+            />
+          </div>
+          <div>
+            <label htmlFor="priceSession">
+              Book session Date picker timezone
+            </label>
+            <select id="timezone" onChange={() => updateTime()}>
+              <option value="(GMT-11:00) Pacific/Midway">
+                (GMT-11:00) Pacific/Midway
+              </option>
+              <option value="(GMT-11:00) Pacific/Niue">
+                (GMT-11:00) Pacific/Niue
+              </option>
+              <option value="(GMT-11:00) Pacific/Pago_Pago">
+                (GMT-11:00) Pacific/Pago_Pago
+              </option>
+              <option value="(GMT-10:00) Pacific/Honolulu">
+                (GMT-10:00) Pacific/Honolulu
```

Comment:
can we make component of it? and same for days and date? as this file code keeep growing. It would be great if we have small components.